package.nls.json

```diff
@@ -174,8 +174,8 @@
 	"command.pr.refreshChanges.title": "Refresh",
 	"command.pr.configurePRViewlet.title": "Configure...",
 	"command.pr.deleteLocalBranch.title": "Delete Local Branch",
-	"command.pr.signin.title": "Sign in to GitHub",
-	"command.pr.signinenterprise.title": "Sign in to GitHub Enterprise",
+	"command.pr.signin.title": "Sign into GitHub",
+	"command.pr.signinenterprise.title": "Sign into GitHub Enterprise",
```

Comment:
There's a difference of opinion about this... I personally favor `into` but some argue that `sign-in` is some sort of magical verb that doesn't hyphenate (verbs don't...) but justifies `Sign in` + `to GitHub` -- which doesn't really make sense to me as I don't `GitHub` as a verb. I treat GitHub as a destination that I can go _into_.

https://github.com/orgs/community/discussions/68753#discussion-5685029
https://code.visualstudio.com/docs/sourcecontrol/github
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization

-- that said, I'm happy to drop this or anything else.